app/services/analytics.ts

```diff
@@ -22,6 +23,8 @@ export default class Analytics extends Service {
         // Wait until everything has settled
         yield waitForQueue('destroy');
 
+        debug(`Track Page: pagePublic: ${pagePublic}, resourceType: ${resourceType}`);
```

Comment:
So we can see when the analytics happen without going to the timeline? Handy! Should we do the same thing for click, or don't bother because we'll be replacing it out soon?

Reply:
I thought about adding it for click too, but probably not worth it since @aaxelb is refactoring it completely.